### ISSUE TITLE
Recently extinct species

### ISSUE THREAD

**Issue by JelleZijlstra** [mammaldiversity/mammaldiversity.github.io#25]
In order to harmonize our databases better I am adding a "recently_extinct" category to Hesperomys. I noticed two possible issues with MDD's treatment of extinct species:

* _Antillomys rayi_ is included in MDD, but it is not on the Red List and as far as I can tell there is no evidence it survived past 1500. (It probably did, like other oryzomyines from the Caribbean, but then why not also include _Megalomys camerhegne_?
* _Archaeolemur edwardsi_ is included in MDD but is not on the Red List, and again I'm not aware of evidence it lived past 1500. The MDD account https://www.mammaldiversity.org/explore.html#genus=Archaeolemur&species=edwardsi&id=1000971 says "it is listed in MSW3 2005", but it isn't.
* _Cordimus hooijeri_, a species I named in 2014, was found in association with _Rattus_ so we hypothesized that it survived past 1499, when the first European contact occurred on Bonaire. There have been some noises about this genus in the literature, but no formal revision. In particular, I'm not confident that this species is really related to the two other species we placed in the genus, from the Pleistocene of Curaçao. Ref: Zijlstra, J.S., McFarlane, D.A., Hoek Ostende, L.W. van den and Lundberg, J. 2014. New rodents (Cricetidae) from the Neogene of Curaçao and Bonaire, Dutch Antilles. Palaeontology 57(5):895-908. doi:10.1111/pala.12091

**Comment by connorjburgin** [comment]
Antillomys rayi – I can’t remember why we included it as a recently extinct species (I think I remember seeing something from a paper saying that it was found with rat skeletons, but I haven’t checked); I’m retaining the species on our list for now; regardless, the name is actually unavailable for the time being, as its description appears in the supplemental material of the paper; its currently being redescribed by the authors in Therya (I reviewed the paper recently).

Megalomys camerhegne – with recently extinct species, I think I’ve been a lot less careful about which species I include and the since 1500 limit is kind of difficult to follow strictly. I’m going to add this species if you feel it is worth doing for consistency.

Archaeolemur edwardsi – I’m pretty confused on why I even included this species in the first place… I might remove it but I’m going to spend some time looking into it a bit more and remove that it was listed in MSW3.

Cordimus hooijeri – will go ahead and add the species as extinct: Zijlstra, J.S., McFarlane, D.A., Hoek Ostende, L.W. van den and Lundberg, J. 2014. New rodents (Cricetidae) from the Neogene of Curaçao and Bonaire, Dutch Antilles. Palaeontology 57(5):895-908. doi:10.1111/pala.12091

If you have any other recently extinct species you think should be included, definitely let us know. I have had a hard time keeping track of them.

**Comment by JelleZijlstra** [comment]
Definitely agree that this is a tricky area since extinction cutoffs are necessarily arbitrary and for many species the date of extinction is not precisely established. Your precise inclusion criteria are obviously up to you, but my thinking is that the purpose of including extinct species is to give a better sense of what the full modern mammal fauna is like, including those species that were lost due to human action over the last few centuries. Therefore, it makes sense to err on the side of including additional species even if their exact extinction date is not firmly established. However, that does obviously mean more work in tracking literature on new species and radiocarbon dating.

In the Caribbean, you (and the Red List) include many species (e.g., _Nesophontes_, many echimyids and oryzomyines) based on subfossils because of evidence that they were found together with European-introduced rodents. I think it's consistent to add _Antillomys_ and _Megalomys camerhogne_ (not _camerhegne_, misspelled it above), even if it's not positively established those species survived past 1500. There are many other extinct species from the Holocene of the Caribbean (sloths, monkeys, bats, a few more rodents), but as far as I know they went extinct earlier. Reference for _camerhogne_ is Mistretta, B.A., Giovas, C.M., Weksler, M. and Turvey, S.T. 2021-04-07. Extinct insular oryzomyine rice rats (Rodentia: Sigmodontinae) from the Grenada Bank, southern Caribbean. Zootaxa 4951(3):434-460. doi:10.11646/zootaxa.4951.3.2

On Madagascar, there are also lots of recently extinct species (_Cryptoprocta spelea_, many lemurs, a few bats and rodents, _Plesiorycteropus_, hippos). You include the fossa, two of the three hippos, and a few lemurs, but not the others. There is some C-14 dating and some rumors that some of these survived into historical times, but I'd have to do more reading to see if there's solid evidence for any others making it past 1500.

**Comment by connorjburgin** [comment]
I agree and that's why I've erred on the side of caution on many of them so far. But with my general focus on living mammals it's been hard to keep track of just recently extinct mammals (my knowledge of mammalian fossil literature is much sparser than that of living species).

**Comment by n8upham** [comment]
For these Caribbean mammals, we summarized a lot of these extinction dates per island (last appearances) in our 2017 Annual Reviews paper: https://doi.org/10.1146/annurev-ecolsys-110316-022754 -- I'll attach a PDF since Fig 2 is helpful visually.  
[Cooke et al. - 2017 - Anthropogenic Extinction Dominates Holocene Declin.pdf](https://github.com/mammaldiversity/mammaldiversity.github.io/files/10542966/Cooke.et.al.-.2017.-.Anthropogenic.Extinction.Dominates.Holocene.Declin.pdf)

I'll also attach a CSV file from the Dryad download (https://datadryad.org/stash/dataset/doi:10.5061/dryad.tj4p5) that has those dates +/- error (sigma)
[fossil.csv](https://github.com/mammaldiversity/mammaldiversity.github.io/files/10542967/fossil.csv)

**Comment by n8upham** [comment]
Antillomys rayi is listed as 
674 +/- 54 ybp according to MacPhee & Flemming 1999 -- which incidentally has a lot of other info on the last 500 years of extinctions: https://www.amnh.org/content/download/232397/4027716/file/Requiem%20Aeternam,%20The%20Last%20Five%20Hundred%20Years%20of%20Mammalian%20Species%20Extinctions.PDF
